Storage/Storage/Model/WooCommerce.xcdatamodeld/Model 31.xcdatamodel/contents

```diff
@@ -0,0 +1,519 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
I noticed that the "Delete Rule" of `downloads` is set to "nullify", but should it be set to "Cascade"? 
What do you think we should use in this case @Ecarrion @jaclync @shiki?

Reply:
**So far what I have tested:**
The `downloadID` of each `ProductDownload` entity is unique, even if the `name` and `fileURL` are same. If you add a downloadable file with the same name and same URL on a single product or multiple products, the `downloadID` is unique everywhere. This proves that a single downloadable file belongs to only one product.

In my opinion, when we are deleting a `Product` from CoreData, all of its downloadable files need to be deleted. So, to me, `Cascade ` makes more sense.

Please rectify me if I am wrong or miss anything.

Reply:
I agree with you, `Cascade` makes more sense because we don't want to continue to store `ProductDownload` entities when the `Product` in relation to one of these entities was deleted. I'd say to change it to `Cascade`, unless there is a valid reason to keep it set to `nullify` (and as of today, it doesn't seem like there is).

Reply:
Updated to `Cascade `

Reply:
yea "cascade" makes sense to me, since each download is unique to a product (it can't be shared with other products) so we can just delete them when a product is deleted 👍

Reply:
Thank you both! 💯